apps/trustlab/src/payload/blocks/Spotlight.js

```diff
@@ -0,0 +1,15 @@
+import RelationshipOverview from "./RelationshipOverview";
+
+const Spotlight = RelationshipOverview(
+  "spotlight",
+  "/images/cms/blocks/spotlight.png",
+  {
+    relationTo: ["helplines", "posts", "resources"],
```

Comment:
```suggestion
    collections: ["helplines", "posts", "resources"],
  {
```
Seems like `collections`/`relationTo` must be specified? Everything in `{}` is pretty much optional.